docs/configuration/http_conn_man/route_config/rate_limits.rst

```diff
@@ -79,20 +91,16 @@ Request Headers
 
 header_name
   *(required, string)* The header name to be queried from the request headers and used to
-  populate the descriptor value for the *descriptor_key*.
+  populate the descriptor entry value for the *descriptor_key*.
```

Comment:
The header name to be queried from the request headers. The header's value is used to populate the value of the descriptor entry for the *descriptor_key*.